src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -363,13 +363,13 @@ private void Initialize()
                 }
 
                 int fileDescriptor = Interop.open(GpioMemoryFilePath, FileOpenFlags.O_RDWR | FileOpenFlags.O_SYNC);
-                if (fileDescriptor < 0)
+                if (fileDescriptor == -1)
```

Comment:
you seem to have merge conflict with yourself, rebasing should auto-fix it

Reply:
@krwq  Rebased... and looks ok. Tests have failed though. I am not convinced the test failure is anything to do with this PR however.

p.s. as a newbie all this git stuff is a black box. is there documentation anywhere on how the project workflows ?

Reply:
I think your rebase should be a bit different:
make sure you use your remote name which points to `github.com/dotnet/iot` (`git remote -v`). Assuming you remote name is `origin`:

```
git fetch --all
git rebase origin/master
```

you will probably see some issues at this point, you will need to use `git mergetool` or some notepad to fix the issues. Might be easier to:

```
git clean -fdx
git reset --hard origin/master
# redo your change by hand
git commit -am "Fix ARM64 issue"
git push RemoteNameWithYourFork -f
```